Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/08-RecordMeeting/ImplementingTimer-04-code-0011.swift

```diff
@@ -45,14 +46,15 @@ struct RecordMeetingTests {
     await clock.advance(by: .seconds(1))
     await store.receive(\.timerTick) {
       $0.secondsElapsed = 4
-      $0.syncUp.meetings.insert(
-        Meeting(
-          id: UUID(0),
-          date: Date(timeIntervalSince1970: 1234567890),
-          transcript: ""
-        ),
-        at: 0
-      )
+      $0.$syncUp.withLock {
+        $0.meetings = [
+          Meeting(
+            id: UUID(0),
+            date: Date(timeIntervalSince1970: 1234567890),
+            transcript: ""
+          )
+        ]
+      }
```

Comment:
📓 I've slightly change the assertion by matching the exact array of meeting to match your recommandation to test against the real result instead of mimicking the code in production.